### ISSUE TITLE
Phonon Pdos projections

### ISSUE THREAD

**Issue by AndresOrtegaGuerrero** [mikibonacci/aiidalab-qe-vibroscopy#47]
The experimentalis are interest to have different projections for the phonon pdos. 
For example to select some atoms, and obtained the contribution of those atoms in pdos

**Comment by mikibonacci** [comment]
Up to now we group with respect to the elements, but we can avoid to sum over the same elements.
We can combine this with the widget by Xing to have the atom selection

**Comment by AndresOrtegaGuerrero** [comment]
Yes, We need to address the issue of phonopy of using atoms with different labels, like 'C1' , 'Fe2" , perhaps we should open an issue in phonopy, and/or try to fix here in the plugin
